src/components/manifold-invoices/manifold-invoices.tsx

```diff
@@ -68,98 +76,193 @@ export class ManifoldInvoices {
 
         return (
           <div>
-            <button onClick={this.select()}>Back to all invoices</button>
+            <button class="ManifoldInvoices__Button" onClick={this.select()}>
+              <span class="ManifoldInvoices__Icon" innerHTML={svgChevronLeft} /> Back to all
+              invoices
+            </button>
             <h1 class="ManifoldInvoices__Subheading">
               {new Intl.DateTimeFormat('en', {
                 year: 'numeric',
                 month: 'long',
               }).format(new Date(invoice.node.end))}
             </h1>
-            <table>
-              <thead class="ManifoldInvoices__TableHeading">
-                <tr class="ManifoldInvoices__TableHeading--GrayLighter">
-                  <th>Service</th>
-                  <th>Due</th>
-                  <th>Duration</th>
-                  <th>Plan</th>
-                </tr>
-              </thead>
-              <tbody>
-                {invoice.node.lineItems.edges.map((edge) => [
-                  <tr>
-                    <td>{edge.node.resource.displayName}</td>
-                    <td>{$(edge.node.cost)}</td>
-                    <td>{edge.node.duration}</td>
-                    <td>{edge.node.resource.plan.displayName}</td>
-                  </tr>,
-                  edge.node.subLineItems && edge.node.subLineItems.edges.length && (
-                    <tr>
-                      <td colSpan={4}>
-                        <table>
-                          <thead>
-                            <tr>
-                              <th>Feature(s)</th>
-                              <th>Cost</th>
-                              <th>Usage</th>
-                            </tr>
-                          </thead>
-                          <tbody>
-                            {edge.node.subLineItems.edges.map((sub) => (
-                              <tr>
-                                <td class="ManifoldInvoices__TableCell--GrayDark ManifoldInvoices__TableCell--Caps">
-                                  {sub.node.item}
-                                </td>
-                                <td class="ManifoldInvoices__TableCell--GrayDark">
-                                  {$(sub.node.cost)}
-                                </td>
-                                <td class="ManifoldInvoices__TableCell--GrayDark">
-                                  {sub.node.description}
-                                </td>
-                              </tr>
-                            ))}
-                          </tbody>
-                        </table>
-                      </td>
-                    </tr>
+            <div role="table" class="ManifoldInvoices__Table ManifoldInvoices__Table--Details">
```

Comment:
I broke away from using a `<table>` because a) we needed a slightly-different markup format, b) this uses CSS Grid anyway, and c) if we want to sort / have responsive styles we’d have to use different elements anyway.

Reply:
makes sense. and bonus points for adding the proper `role`s